cron/update_lunch.py

```diff
@@ -1,31 +1,38 @@
 import datetime
 import logging
+import os
+
+from google.cloud import ndb
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "service_account.json"

```

Comment:
Do you need this line? It's being set in line 232 already, I think.

Reply:
for now it avoids an error